dev-environment.yml

```
@@ -0,0 +1,11 @@
+name: libsemigroups_pybind11_dev
+
+channels:
+  - conda-forge
+
+dependencies:
+  - fmt
+  - pip: 
+    - accepts
+    - bs4
+    - lxml
```

I checked conda-forge and I think we can do this in the interest of minimising conda-pip interactions
```suggestion
dependencies:
  - fmt
  - bs4
  - lxml
  - pip: 
    - accepts
```